source/develop/developer-guide/vdsm/developers.html.md

```diff
@@ -18,12 +18,24 @@ wiki_last_updated: 2015-08-26
 To build VDSM, enable the oVirt repositories by installing the ovirt-release rpm:
 
       yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
-
+      
+      
+ or if you are using Fedora 
+ 
+ 
+      dnf install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
```

Comment:
This sounds a bit repetitive to me. How about asking to set `YUM=yum` or `YUM=dnf` depending on operating system, and then use `$YUM`?

Is it too technical for the casual admin?